.github/workflows/test.yml

```diff
@@ -122,7 +122,7 @@ jobs:
       matrix:
         python-version: ["3.9", "3.10", "3.11"]
         airflow-version: ["2.4", "2.5", "2.6", "2.7", "2.8", "2.9", "2.10", "2.11", "3.0"]
-        dbt-version: ["1.9"]
+        dbt-version: ["1.9", "1.10"]
```

Comment:
@pankajkoti I noticed that ATM we're running too many integrations tests against BQ (async) and that we're hitting the BQ limit. Do you think it would make sense for us to keep just 1.10 in this part? Ideally we'd reduce a bit the matrix to limit the combinations we're testing against.

Reply:
makes sense. Let's try just 1.10

Reply:
Done in 03d36b04b62563277aeb9b26826a7e2d48eb15b5